scrapers/bondageliberation.yml

```diff
@@ -0,0 +1,44 @@
+name: bondageliberation
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - bondageliberation.com/videos
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    common: &commonSelectors
+      $dtBase: //div[@class="flex justify-between mt-2 cursor-pointer"]/div[1]
+      $dptBase: //div[@class="mt-5 info-content "]
+    scene:
+      Studio: 
+        Name: //title/text()
+      Title: 
+        selector: $dtBase/span[1]/text()
+      Date:
+        selector: $dtBase/span[2]/text()
+        postProcess:
+          - parseDate: 2006-01-02 15:04:05
+      Details:
+        selector: $dptBase/div[2]/pre/text()
+        concat: "\n\n"
+      Performers: 
+        Name: 
+          selector: $dptBase/div[4]//a/text()
+          postProcess:
+            - replace:
+              - regex: ",$"
+                with: ""
+      Tags:
+        Name: $dptBase/div[3]//span/text()
+      Image: 
+        selector: //div[@class="w-full h-full flex items-center absolute justify-center top-0 "]/@style
+        postProcess:
+          - replace:
+            - regex: "background-image: url\\(\"(.*)\"\\); filter: blur\\(80px\\);"
+              with: $1     
```

Comment:
In general we try to avoid positional selectors if possible
For this case using as a common 
```yaml
    common:
      $info: //div[contains(@class,"info-content")]
```
we can get the below selectors
```yaml
      Title: 
        selector: $info//span[contains(@class,"title-content")]
      Date:
        selector: $info//span[contains(@class,"text-grey-davos")]
        postProcess:
          - parseDate: 2006-01-02 15:04:05
      Details:
        selector: $info//pre[contains(@class,"description-content")]/text()
        concat: "\n\n"
      Performers: 
        Name: 
          selector: //div[@class="info-casts-title"]/following-sibling::div/a
          postProcess:
            - replace:
              - regex: ",$"
                with: ""
      Tags:
        Name: $info//a[contains(@class,"tag")]/span
      Image: 
        selector: //div[contains(@style,"blur(80px)")]/@style
        postProcess:
          - replace:
            - regex: 'background-image: url\("(.*)"\); filter: blur\(80px\);'
              with: $1     
```
that are simpler and do not rely on positions, complex classes

They also seem to work ok ? with a few scenes i tested

Reply:
The image selector doesn't work anymore. It seems they are embedding the image via an external script via cloudfront.net <https://d26ap12j9jlhjd.cloudfront.net/static/js/main.def3d733.js>